README.md

```diff
@@ -1,2 +1,9 @@
 # PRS-UI
 Creating a UI for the previous Javascript Paper rock scissors game uploaded
+---
+-Create a user interface for the previous uploaded javascript paper rock scissors game
+-Use buttons for user inputs
+-The first player to score five points should be decared winner
+-Display the winner in a div usind DOM methods
+-Display the scores for each round.
+-Use DOM manipulation for the UI.
```

Comment:
- Since we have a proper project using more than 1 technology to implement. it would be nice we start adding proper documentation to our project. You can check this [template](https://github.com/othneildrew/Best-README-Template), what you think?😉